pyproject.toml

```
@@ -160,6 +160,7 @@ namespace_packages = true
 [[tool.mypy.overrides]]
 module = [
     "*.hatchling.*",
+    "*.hatch.config.*",
     "*.hatch.utils.*",
```

I found that here should be:
```suggestion
    "hatch.config.*",
    "hatch.utils.*",
```